docs/DeveloperGuide.md

```diff
@@ -555,7 +556,7 @@ The User profile will be displayed on the profile panel should there be any exce
 </div>
 
 **Parsing the inputs** - When the user enters the input, the `ViewCommandParser` will check if the arguments are empty first.
-If it is not empty, then `ViewCommandParser` will extract try to find tokens that were prefixed `/n`. If a name is not present, it will search for an index (of `int` type) instead. 
+If it is not empty, then `ViewCommandParser` will extract try to find tokens that were prefixed `n/`. If a name is not present, it will search for an index (of `int` type) instead. 
```

Comment:
Some suggested changes:
- When user input is received, the `ViewCommandParser` will first check if the arguments are empty.
- If the arguments are not empty, the `ViewCommandParser` will try to extract tokens that are prefixed with`n/`. 
- If a name is not present, it will search for an index (of type`int`) instead.

Reply:
@zichen-3974 Good idea 👍 Thanks for the suggestion!